scripts/superdesk-highlights/highlights.js

```diff
@@ -65,6 +66,12 @@
             }
         };
 
+        function setLabel(objItems) {
+            _.forEach(objItems, function (item) {
+                item.label = item.desks.length ? item.name : item.name + gettext(' (Global)');
```

Comment:
for the sake of translation, might be better to have it like: `item.name + ' ' + gettext('(Global)')` so that if someone omits the space while translating this will still work